src/app/components/navbar/navbar.component.html

```diff
@@ -41,30 +79,60 @@
 						<a class="dropdown-item waves-effect waves-light" routerLink="/payment/saved-card">Saved Card</a>
 				</div>
 			</li>
 
       <li *ngIf="authenticated" class="nav-item avatar dropdown">
-        <a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle waves-effect waves-light"
-          data-toggle="dropdown" id="navbarDropdownMenuLink-6">
+        <a
+          aria-expanded="false"
+          aria-haspopup="true"
+          class="nav-link dropdown-toggle waves-effect waves-light"
+          data-toggle="dropdown"
+          id="navbarDropdownMenuLink-6"
+        >
           <span class="profile-ava">
-            <img [src]="profileImage" alt="avatar image" class="rounded-circle z-depth-0" height="35">
+            <img
+              [src]="profileImage"
+              alt="avatar image"
+              class="rounded-circle z-depth-0"
+              height="35"
+              alt=""
+            />
           </span>
-          <span class="username">{{firstName}} {{lastName}}</span>
+          <span class="username">{{ firstName }} {{ lastName }}</span>
           <b class="caret"></b>
         </a>
-        <div aria-labelledby="navbarDropdownMenuLink-6" class="dropdown-menu dropdown-menu-lg-right dropdown-secondary">
-          <a class="dropdown-item waves-effect waves-light eborder-top" routerLink="/profile"
-            routerLinkActive="active"><i class="fa fa-user"></i> My Profile</a>
-          <a class="dropdown-item waves-effect waves-light" routerLink="/create-company"><i class="fa fa-plus"></i>
-            Create Company</a>
-          <a class="dropdown-item waves-effect waves-light" routerLink="/reset-link"><i class="fa fa-plus"></i> Password
-            Reset</a>
-          <a (click)="handleLogout()" class="dropdown-item waves-effect waves-light profileDropDown" routerLink="/logout"
-            routerLinkActive="active">
-            <i class="fa fa-sign-out "></i> Log Out</a>
+        <div
+          aria-labelledby="navbarDropdownMenuLink-6"
+          class="dropdown-menu dropdown-menu-lg-right dropdown-secondary"
+        >
+          <a
+            class="dropdown-item waves-effect waves-light eborder-top"
+            routerLink="/profile"
+            routerLinkActive="active"
+            ><i class="fa fa-user"></i> My Profile</a
+          >
+          <a
+            class="dropdown-item waves-effect waves-light"
+            routerLink="/create-company"
+            ><i class="fa fa-plus"></i> Create Company</a
+          >
+          <a
+            class="dropdown-item waves-effect waves-light"
+            routerLink="/reset-link"
+            ><i class="fa fa-plus"></i> Password Reset</a
+          >
+
+          <a
+            (click)="handleLogout()"
+            class="dropdown-item waves-effect waves-light"
+            routerLink="/logout"
+            routerLinkActive="active"
+          >
+            <i class="fa fa-sign-out"></i> Log Out</a
+          >
```

Comment:
@patrickf949  Check the navbar carefully, you have overridden an existing styling.

Reply:
Thanks done.